README.md

```diff
@@ -1,81 +1,49 @@
-| Scorecard |https://github.com/ossf/scorecard | Best Practices WG | Applying for [Incubating](process/project-lifecycle-documents/repository_service_for_tuf_incubation_stage.md) |[Security Baseline - Once Sandbox](https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---once-sandbox) <br /> and <br />[Security Baseline - To Become Incubating](https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---to-become-incubating)<br /> and <br />[Security Baseline - Once Incubating](https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---to-become-incubating) |
- 
-## Get Involved
-
-*   Official communications occur on the [Security Baseline Mailing List](https://lists.openssf.org/g/openssf-sig-security-baseline). 
-*   [Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups).
-*   [Join our Slack](https://app.slack.com/client/T019QHUBYQ3/C07DC6TT2QY)
-
-
-### Quick Start
-
-*   Need contributors to improve documents, raise awareness of the initiative and increase adoption of the security baseline.
-*   Need contributors to evangelize the security baseline for Linux Foundation wide adoption .
-*   Create an issue or submit a OR to turn your ideas into reality
-
-### Meeting times
-*   Every other Tuesday @ 10:00am EST on [zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/97740884759?password=5cab7229-2324-4816-81db-517812a088a9)
-*   [Meeting Minutes](https://docs.google.com/document/d/16tL1Ln7owIRXSoCKgyYHCs9-JP9iw-ouyk8koGAeHA0/edit)
-
-## Governance
-
-[TODO: Update this link to your specific CHARTER.md file]
-The [CHARTER.md](https://github.com/ossf/project-template/blob/main/CHARTER.md) outlines the scope and governance of our group activities.
-
-*   Lead name: Eddie Knight 
-*   Co-Lead name: Michael Lieberman
-
-## Intellectual Property
-
-In accordance with the [OpenSSF Charter (PDF)](https://charter.openssf.org/), work produced by this group is licensed as follows:
-
-1. Software source code
-* Apache License, Version 2.0, available at https://www.apache.org/licenses/LICENSE-2.0;
-
-## Antitrust Policy Notice
+# Open Source Project Security Baseline
+
+The Open Source Project Security Baseline (OSPS Baseline) is designed to act as a minimum definition of requirements for a project relative to it's maturity level.
+
+All definitions are maintained in YAML format for tandem machine and human readibility.
+
+## Baseline Structure
+
+Each entry has the following values:
+
+- **ID**:
+  - Entries 1-39 are reserved for maturity level 1
```

Comment:
Approving since this is current state, but the ID ranges need to be removed.

Reply:
objective noted, thank you